worldengine/cli/main.py

```diff
@@ -441,10 +450,10 @@ def main():
     if args.temps and len(args.temps.split('/')) is not 6:
         usage(error="temps must have exactly 6 values")
 
-    temps =[.874, .765, .594, .439, .366, .124]
+    temps = [.874, .765, .594, .439, .366, .124]
     if args.temps:
         temps = args.temps.split('/')
-        for x in range(0,6):
+        for x in range(0,len(temps)):
```

Comment:
We do test the length of the array. We don't test order because there is a very slight possibility someone will want to input a series where parameter X is less than parameter X-1 (this would result in a zone being skipped).